packages/app-extrinsics/src/Balance.tsx

```diff
@@ -30,4 +30,9 @@ class BalanceDisplay extends React.PureComponent<Props> {
   }
 }
 
-export default withCall('query.balances.freeBalance', { paramName: 'params' })(BalanceDisplay);
+export default withMulti(
```

Comment:
Not sure we need a withMulti here? I am guessing some types started complaining.

Reply:
Yes, some of the reactive components threw issues when exported as withCalls(...) rather than withCall() or withMulti() - 'params' isn't supported?

Type '{ params: string | undefined; }' is not assignable to type 'IntrinsicAttributes & Pick<Props, "className" | "style" | "children" | "label" | "callResult" | "callUpdated" | "callUpdatedAt" | "accounts_idAndIndex"> & { children?: ReactNode; }'. Property 'params' does not exist on type 'IntrinsicAttributes & Pick<Props, "className" | "style" | "children" | "label" | "callResult" | "callUpdated" | "callUpdatedAt" | "accounts_idAndIndex"> & { children?: ReactNode; }'.